samples/net/openthread/coap/prj.conf

```diff
@@ -26,3 +26,7 @@ CONFIG_OPENTHREAD_XPANID="11:11:11:11:22:22:22:22"
 CONFIG_OPENTHREAD_NETWORKKEY="00:11:22:33:44:55:66:77:88:99:aa:bb:cc:dd:ee:ff"
 CONFIG_OPENTHREAD_CHANNEL=15
 CONFIG_OPENTHREAD_NETWORK_NAME="OpenThreadDemo"
+
+# Use ZMS as storage backend for Secure Storage
+CONFIG_SECURE_STORAGE_ITS_STORE_IMPLEMENTATION_ZMS=y
```

Comment:
Wouldn't we need this in other samples as well? And those generic networking samples that have `overlay-ot.conf`?

Reply:
This is still no clear to me, do we need thsoe configs in other samples/OT overlays as well?

Reply:
I added the DNM label to ensure that this PR is not merged accidentally before we're happy with test coverage